SmoothScrolling/Client/CollectionView/CollectionView/UserCell.swift

```diff
@@ -17,16 +17,11 @@ class UserCell: UICollectionViewCell {
 
     fileprivate var downloadTask: URLSessionDataTask?
     
-    override init (frame: CGRect) {
-        super.init(frame: frame)
-        
+    override func awakeFromNib() {
+        super.awakeFromNib()
         avatar.setRoundedImage(UserCell.defaultAvatar)
```

Comment:
A further optimization would be to move setOpaqueBackground inside awakeFromNib().